pyproject.toml

```diff
@@ -21,6 +21,7 @@ license = "Apache-2.0"
 
 [tool.poetry.dependencies]
 python = ">=3.8"
+appstoreconnect = { git = "https://github.com/mahangu/appstoreconnectapi.git", branch = "fix-dependencies" }
```

Comment:
We have [forked `appstoreconnect`](https://github.com/mahangu/appstoreconnectapi/tree/fix-dependencies) to fix a PyJWT dependency conflict it has with `singer_sdk`:

```

Because no versions of singer-sdk match >0.36.1,<0.37.0
 and singer-sdk (0.36.1) depends on PyJWT (>=2.4,<3.0), singer-sdk (>=0.36.1,<0.37.0) requires PyJWT (>=2.4,<3.0).
And because appstoreconnect (0.10.1) depends on PyJWT (>=1.6.4,<2.dev0)
 and no versions of appstoreconnect match >0.10.1,<0.11.0, singer-sdk (>=0.36.1,<0.37.0) is incompatible with appstoreconnect (>=0.10.1,<0.11.0).
So, because tap-appstore depends on both appstoreconnect (~=0.10.1) and singer-sdk[testing] (~=0.36.1), version solving failed.
```